imod/tests/test_mf6/test_mf6_simulation.py

```diff
@@ -53,6 +54,40 @@ def test_circle_roundtrip(circle_model, tmpdir_factory):
     roundtrip(circle_model, tmpdir_factory, "circle")
 
 
+@pytest.mark.usefixtures("twri_model")
+def test_twri_gdal(twri_model, tmpdir_factory):
```

Comment:
maybe this test should be skipped if rioxarray is not installed.

Reply:
A development install always includes rioxarray, so skipping this is unnecessary.